bundles/com.salesforce.bazel-java-sdk/aspect/intellij_info_impl.bzl

```diff
@@ -971,23 +971,19 @@ def intellij_info_aspect_impl(target, ctx, semantics):
     direct_exports = []
     if JavaInfo in target:
         direct_exports = collect_targets_from_attrs(rule_attrs, ["exports"])
-        transitive_exports = target[JavaInfo].transitive_exports
-        export_deps = [
-            make_dep_from_label(label, COMPILE_TIME)
-            for label in transitive_exports.to_list()
-        ]
+        export_deps.extend(make_deps(direct_exports, COMPILE_TIME))
```

Comment:
This change will get wiped out during the next copy of the Intelli aspect. We need to have a code comment here to show that this section has diverged intentionally from upstream (or did you get this fix from upstream?). Also in the note should explain the use case that requires the fix so we can retest after the next update.

Reply:
See the commit message:
https://github.com/salesforce/bazel-eclipse/pull/406/commits/6a3a74a1724e9792f26a9a6d1fc71cfea2383a90

Reply:
OK great.